qiskit/qasm/node/gate.py

```diff
@@ -57,11 +57,11 @@ def n_bits(self):
         """Return the number of qubit arguments."""
         return self.bitlist.size()
 
-    def qasm(self, prec=15):
```

Comment:
Because this API is exposed to the user, I think the parameter should be deprecated.